package.json

```diff
@@ -4,6 +4,8 @@
   "private": true,
   "dependencies": {
     "@ant-design/icons": "^4.5.0",
+    "@davatar/react": "^1.8.1",
+    "@ethersproject/providers": "^5.5.2",
```

Comment:
We have web3 and our bundle is unfortunately already big! can we utilize the existing web3js instead?
I didn't find an alternative for `getDefaultProvider`! 

I want to suggest two options:
1. Using some reliable ens subgraph by which we will have less trouble I think
2. Switching to web3 and add an mainnet nodeurl to our setting if needed

Reply:
@aminlatifi Sadly web3js dont have ens support :(

Reply:
> @aminlatifi Sadly web3js dont have ens support :(

Yeah, I understood the same!

Reply:
@aminlatifi I think we can reduce it in further iterations, we really love to see this app on ens.domains homepage

once this is available in prod we will feature this app in https://ens.domains homepage and tweet about it

Reply:
@aminlatifi can we move this forward or something to look later?

Reply:
> 

Yeah, I will merge it.

Reply:
Love to feature u folks on ens.domains homepage